test/tests/functional/pbs_hook_timeout.py

```diff
@@ -71,9 +70,12 @@ def setUp(self):
         self.server.manager(MGR_CMD_CREATE, NODE, id=self.hostB)
 
         self.server.manager(MGR_CMD_CREATE, NODE, id=self.hostC)
-
+        for mom in self.moms.values():
+            self.server.expect(NODE, {'state': 'free'}, id=mom.shortname)
+        """
         self.server.expect(VNODE, {'state=free': 3}, op=EQ, count=True,
                            max_attempts=10, interval=2)
+        """
```

Comment:
this is commented out. if we decide to check for "all" nodes to be in free state, we should remove the commented out code.

Reply:
Removed.